Sources/MessagesViewController.swift

```diff
@@ -82,6 +84,13 @@ open class MessagesViewController: UIViewController {
             addKeyboardObservers()
             messagesCollectionView.contentInset.bottom = messageInputBar.frame.height
             isFirstLayout = false
+            
+            //scroll to bottom at first load
+            guard scrollsToBottomOnFirstLayout == true else {
+                return
+            }
+            scrollsToBottomOnFirstLayout = false
```

Comment:
let's not modify the bool after we scroll to bottom, all we're really concerned with is the `isFirstLayout` flag